problem_builder/swipe.py

```diff
@@ -60,9 +60,9 @@ class SwipeBlock(SubmittingXBlockMixin, StudentViewUserStateMixin, Questionnaire
     )
 
     student_choice = String(
-        # {Last input submitted by the student
         default="",
         scope=Scope.user_state,
+        help=_("Last input submitted by the student.")
     )
 
     correct_choice = String(
```

Comment:
This should just be a `Boolean` called `correct`, and it should not have a `values_provider`.

Reply:
:+1: